portfolio/src/main/webapp/index.html

```diff
@@ -9,8 +9,11 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
     <script src="script.js"></script>
+    <script async defer 
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCceMyx3SzOJO2G1yGVF8xY52vQFV5B5ck&callback=initMap">
+    </script>
   </head>
-  <body onload="renderForm()">
+  <body onload="renderForm();initMap();">
```

Comment:
I'd recommend putting these in an event listener for something like [`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) in your script to separate content from behavior (https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns);